lib/model/channel.dart

```diff
@@ -168,6 +168,11 @@ class ChannelStoreImpl with ChannelStore {
 
       case SubscriptionRemoveEvent():
         for (final streamId in event.streamIds) {
+          assert(streams.containsKey(streamId)
+            && streams[streamId] is Subscription);
```

Comment:
We could break the `&&`'s into their own lines, so it is clearer when only one of them fails.